options_linux.cmake

```
@@ -91,14 +93,33 @@ if (DESKTOP_APP_USE_ALLOCATION_TRACER)
     )
 endif()
 
-if (DESKTOP_APP_USE_PACKAGED)
-    set(THREADS_PREFER_PTHREAD_FLAG ON)
-    find_package(Threads REQUIRED)
-    target_link_libraries(common_options
+set(CMAKE_REQUIRED_LINK_OPTIONS -pthread)
+check_cxx_source_compiles("
```

By the way, you can use multiline strings here if you find them prettier.
https://cmake.org/cmake/help/v3.24/manual/cmake-language.7.html#bracket-argument

```suggestion
check_cxx_source_compiles([[
```